mobile-app/app/components/icons/assets/_Default.tsx

```diff
@@ -1,7 +1,78 @@
+import { RootState } from '@store'
+import { tokenSelectorByDisplaySymbol } from '@store/wallet'
 import randomColor from 'randomcolor'
-import Svg, { Circle, SvgProps, Text } from 'react-native-svg'
+import Svg, { Circle, SvgProps, Text, G, Path } from 'react-native-svg'
+import { useSelector } from 'react-redux'
 
 export function _Default (symbol: string): (props: SvgProps) => JSX.Element {
+  const tokenDetail = useSelector((state: RootState) => tokenSelectorByDisplaySymbol(state.wallet, symbol))
+
+  if (tokenDetail?.isLoanToken === true) {
+    return function (props: SvgProps): JSX.Element {
+      const name = symbol.substring(1, 5).toUpperCase()
+      return (
```

Comment:
Create a react component for this, don't inline functions

Reply:
Will create separate component for this.